cinder/volume/drivers/nexenta/nfs.py

```diff
@@ -125,44 +123,40 @@ def __init__(self, *args, **kwargs):
 
     @staticmethod
     def get_driver_options():
-        return (
-            options.NEXENTA_CONNECTION_OPTS +
-            options.NEXENTA_NFS_OPTS +
-            options.NEXENTA_DATASET_OPTS
-        )
+        return options.NEXENTASTOR4_NFS_OPTS
 
     def do_setup(self, context):
         self.nms = jsonrpc.NmsProxy(self.driver_volume_type,
-                                    self.root_path,
+                                    self.nas_path,
                                     self.configuration)
 
     def check_for_setup_error(self):
         """Check root filesystem, NFS service and NFS share."""
-        if not self.nms.folder.object_exists(self.root_path):
+        if not self.nms.folder.object_exists(self.nas_path):
             message = (_('NFS root filesystem %(path)s not found')
-                       % {'path': self.root_path})
+                       % {'path': self.nas_path})
             raise jsonrpc.NmsException(code='ENOENT', message=message)
-        filesystem = self.nms.folder.get_child_props(self.root_path, '')
-        if filesystem['mountpoint'] == 'none':
+        self.nas_root = self.nms.folder.get_child_props(self.nas_path, '')
```

Comment:
same as above

Reply:
Changed to self.nas_stat and self.san_stat, thank you!